app/Services/StandService.php

```diff
@@ -137,4 +139,55 @@ private function standExists(int $standId): bool
     {
         return Stand::where('id', $standId)->exists();
     }
+
+    /**
+     * Delete a given stand
+     *
+     * @param string $airfield
+     * @param string $identifier
+     * @throws \Exception
+     */
+    public function deleteStand(string $airfield, string $identifier)
+    {
+        $stand = Stand::with('airfield')->whereHas(
+            'airfield',
+            function (Builder $query) use ($airfield) {
+                $query->where('code', $airfield);
+            }
+        )->where('identifier', $identifier)
+            ->first();
+
+        if (!$stand) {
```

Comment:
Maybe a candidate for a find or fail?

Reply:
Was toying with that - decided against it as this'll be mainly used in migrations and we probably wouldn't want to crash the entire migration over something not being deleted when it never existed anyway